src/frontend/templates/pages/account.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+{% load custom_formatting %}
+{% load custom_translation %}
+
+{% block title %}{% trans "KataGo - Account" %}{% endblock %}
+
+
+{% block content %}
+
+<h2 class="title">{% trans "KataGo - Account" %}</h1>
+
+{% url 'account_change_password' as password_url %}
+<a href="{{password_url}}">
```

Comment:
Does the password_url exist ?

Reply:
Yes, it is defined by django allauth:
https://django-allauth.readthedocs.io/en/latest/views.html

Reply:
Ah ok :)